.brownie/network-config.yaml

```diff
@@ -75,6 +75,18 @@ live:
         id: bsc-main
         host: https://bsc-dataseed.binance.org
         explorer: https://api.bscscan.com/api
+  - name: Evmos
+    networks:
+      - name: Testnet
+        chainid: 9000
+        id: evmos-test
+        host: https://eth.bd.evmos.dev:8545
+        explorer: https://evm.evmos.dev/
+      - name: Mainnet
+        chainid: 9001
+        id: evmos-mainnet
```

Comment:
should be `evmos-main` in order to be consistent with other entries

Reply:
resolved in latest commit